app/src/main/kotlin/com/wire/android/ui/home/settings/about/licenses/LicensesScreen.kt

```diff
@@ -55,9 +64,34 @@ fun LicensesContent(
                 .fillMaxSize()
                 .padding(internalPadding)
         ) {
-            LibrariesContainer(
-                modifier = Modifier.fillMaxSize()
+            val context = LocalContext.current
+            val libraries = produceState<Libs?>(null) {
+                value = withContext(Dispatchers.IO) {
+                    Libs.Builder().withContext(context).build()
+                }
+            }
+            val libs = libraries.value?.libraries
```

Comment:
I think this logic should be done in viewModel and then we inject the list of libs.

`LicensesContent` looks like it's `LicensesScreen` we have everyting there. What about putting the scaffold in `LicensesScreen` and `LicensesContent` will just have the content as the names says ?

Reply:
make sense